### DISCUSSION TITLE
Remove agent connection notifications

### DISCUSSION THREAD

**Discussion by marcelonomos** [Ylianst/MeshCentral#5340] [Q&A]
Hello everyone, I need to know if there is any way to remove or inhibit notifications from agents that connect and disconnect from the server.
I use meshCentral to support my software clients and whenever I am connected to a client this notification appears and occasionally I click on it and leave the client I was serving.

![NotficaçõesMeshCentral](https://github.com/Ylianst/MeshCentral/assets/144562830/8c9ef92c-7aa8-4b6f-a194-bf622714582a)

If you could give me this help... that would be great.
Thank you very much

**Comment by si458** [comment]
This notification only shows if you have enabled the device connection/disconnection notifications in a computer group settings like below pic 
![Screenshot_20230909_224335_Firefox.jpg](https://github.com/Ylianst/MeshCentral/assets/765314/8fddd0f6-2008-4346-811f-bf37fcb92f24)

**Comment by si458** [comment]
This notification only shows if you have enabled the device connection/disconnection notifications in a computer group settings like below pic 
![Screenshot_20230909_224335_Firefox.jpg](https://github.com/Ylianst/MeshCentral/assets/765314/8fddd0f6-2008-4346-811f-bf37fcb92f24)

OR

You have enabled the notifications in ur user settings in the my account tab like below pic
![Screenshot_20230909_225238_Firefox.jpg](https://github.com/Ylianst/MeshCentral/assets/765314/ebb9780b-652d-4e88-8f36-55917d5770fa)
